### ISSUE TITLE
graph_from_point with network_type="bike" loses many roads and cycleways

### ISSUE THREAD

**Issue by megstanley** [gboeing/osmnx#253]
Calling graph_from_point in Central London, with and without network_type="bike". Many of the network components that are known to be cycleable are lost.

Ubuntu 18.04.1, Python 3, and OSMnx 0.9

#### Environment

<details>
Package          Version    
---------------- -----------
altair           2.3.0      
asn1crypto       0.24.0     
attrs            18.2.0     
backcall         0.1.0      
boto3            1.9.89     
botocore         1.12.89    
branca           0.3.1      
CacheControl     0.12.5     
certifi          2018.11.29 
cffi             1.11.5     
chardet          3.0.4      
Click            7.0        
click-plugins    1.0.4      
cligj            0.5.0      
cryptography     2.5        
cycler           0.10.0     
DateTime         4.3        
decorator        4.3.2      
descartes        1.1.0      
docutils         0.14       
entrypoints      0.3        
Fiona            1.8.4      
folium           0.7.0      
GDAL             2.4.0      
geographiclib    1.49       
geopandas        0.4.0      
geopy            1.18.1     
idna             2.8        
ipykernel        5.1.0      
ipython          7.2.0      
ipython-genutils 0.2.0      
iso3166          0.9        
jedi             0.13.2     
Jinja2           2.10       
jmespath         0.9.3      
jsonschema       3.0.0a3    
jupyter-client   5.2.4      
jupyter-core     4.4.0      
kiwisolver       1.0.1      
mapbox           0.17.2     
MarkupSafe       1.1.0      
matplotlib       3.0.2      
msgpack          0.6.1      
munch            2.3.2      
networkx         2.2        
numpy            1.16.1     
osmnx            0.9        
palettable       3.1.1      
pandas           0.24.1     
parso            0.3.3      
patsy            0.5.1      
pexpect          4.6.0      
pickleshare      0.7.5      
pip              19.0.1     
polyline         1.3.2      
prompt-toolkit   2.0.8      
psycopg2         2.7.7      
ptyprocess       0.6.0      
pycparser        2.19       
Pygments         2.3.1      
pyOpenSSL        19.0.0     
pyparsing        2.3.1      
pyproj           1.9.6      
pyrsistent       0.14.9     
pysal            2.0.0      
PySocks          1.6.8      
python-dateutil  2.8.0      
pytz             2018.9     
pyzmq            17.1.2     
requests         2.21.0     
Rtree            0.8.3      
s3transfer       0.2.0      
scikit-learn     0.20.2     
scipy            1.2.0      
seaborn          0.9.0      
setuptools       40.7.3     
Shapely          1.6.4.post2
six              1.12.0     
sklearn          0.0        
SQLAlchemy       1.2.17     
statsmodels      0.9.0      
toolz            0.9.0      
tornado          5.1.1      
traitlets        4.3.2      
uritemplate      3.0.0      
urllib3          1.24.1     
vincent          0.4.4      
wcwidth          0.1.7      
wheel            0.32.3     
zope.interface   4.6.0      
</details>

#### Code snippet that independently and completely reproduces the issue

```python
import osmnx as ox
import networkx as nx
import matplotlib.pyplot as plt
%matplotlib inline

point = (51.469058, -0.018071)
localG = ox.graph_from_point(point, distance=500)
localG_bikeonly = ox.graph_from_point(point, distance=500, network_type="bike")

fig1, ax1 = ox.plot_graph(localG,node_size=0, bgcolor='black', edge_color="white", edge_linewidth=0.5, dpi=250)

fig2, ax2 = ox.plot_graph(localG_bikeonly,node_size=0, bgcolor='black', edge_color="white", edge_linewidth=0.5, dpi=250)


```

**Comment by gboeing** [comment]
@megstanley can you provide the OSMIDs of some of the example ways on OpenStreetMap that are cycleable but are not appearing in your graphs? I'll check them out.

**Comment by megstanley** [comment]
I am not able to get hold of them at the moment -- but in the example code I have placed above, the two figures show a big loss of network between the first and the second, where the second is from network_type = bike. However, most of the lost network is regular roads where cycling is definitely permitted.

**Comment by gboeing** [comment]
@megstanley as I'm not familiar with the area, I'd need some OSMIDs of example ways that you believe are incorrectly missing in the bike graph, to do some investigation.

**Comment by megstanley** [comment]
OK, I can give some OSMIDS:
ways
54254170
54254168
which are part of a cycleway and 
651852808
which is a trunk road but has a cycleway shared with a busway.

**Comment by gboeing** [comment]
@megstanley it looks like it's all working properly to me. In your code snippet, you have `retain_all=False` (which is the default parameterization), so OSMnx only retains the largest connected graph component.

If you set `retain_all=True`, you will see those 3 OSMIDs you listed do appear in this graph. However, this graph won't be fully connected, because there are some ways tagged on OSM as "footway" or linking the two primary hemispheres of the biking network.

One last thing I noticed is that OSMnx was incorrectly allowing ways tagged as "steps" into the biking network graph, which has been fixed in 3113509.
